Web/coolprop/LowLevelAPI.rst

```diff
@@ -25,6 +25,46 @@ which yields the output:
 
 .. literalinclude:: snippets/AbstractState1.cxx.output
 
+Here is an example of the shared library usage with Julia wrapper::
+
+    julia> import CoolProp
+
+    julia> PT_INPUTS = CoolProp.get_input_pair_index("PT_INPUTS")
```

Comment:
On Julia 0.4, all the error I tested had adapted error messages.
On Julia 0.3, I still need the ppa to be corrected to have correct error handling.

This is possible that I did not test all the possible errors, but I tried to have the main possible ones (up to my knowledge).

It may be useful to have a list somewhere with the possible strings for the input pair.